frontend/src/libhoclient/host_orchestrator_client.go

```diff
@@ -48,11 +48,26 @@ type ConnectWebRTCOpts struct {
 	LocalICEConfig *wclient.ICEConfig
 }
 
-type BuildAPICredential struct {
+type BuildAPICreds interface {
+	ApplyToHTTPRequest(rb *HTTPRequestBuilder)
+}
+
+type AccessTokenBuildAPICreds struct {
```

Comment:
What about the one that asks the Cloud Orchestrator to inject? Are you going to implement it in the cloud orchestrator project?

Reply:
yes.